docs/install/harvester-configuration.md

```diff
@@ -425,6 +425,20 @@ os:
     mylabel: myvalue
 ```
 
+### `os.sshd.sftp`
+
+#### Definition
+
+`os.sshd` is used to config the sshd. Now, Harvester only supports the `sftp` as the subparameter of `os.sshd`.
```

Comment:
perhaps say set the field to true enable the SFTP subsystem.

Reply:
Add the comment to the example yaml file.
Thanks for the suggestion!